src/main/resources/edu/cornell/kfs/krad/config/CUKRADSpringBeans.xml

```diff
@@ -72,5 +72,7 @@
 
     <bean id="cf.businessObjectAuthorizationService"
           class="edu.cornell.kfs.kns.service.impl.CuBusinessObjectAuthorizationServiceImpl"/>
+          
+   <bean id="cf.kradExceptionIncidentService" class="edu.cornell.kfs.krad.service.impl.CuKualiExceptionIncidentServiceImpl"/>
```

Comment:
It looks like the base financials version of the "cf.kradExceptionIncidentService" bean does not follow the "parentBean" convention. You may have to include the full XML of the base "cf.kradExceptionIncidentService" bean when you customize it here.

Reply:
fixed